scripts/main.bro

```diff
@@ -69,6 +71,8 @@ function zone_by_depth(domain: string, depth: count): string
 function effective_tld(domain: string): string
 	{
 	local depth=1;
+	if ( effective_tlds_4th_level in domain )
+		depth=4;
 	if ( effective_tlds_3rd_level in domain )
```

Comment:
I suspect this should be an else-if now that you added the additional if statement.  Agree?

```suggestion
	else if ( effective_tlds_3rd_level in domain )
```

Reply:
Oh, yes. That makes sense.  It should be an else-if at line 88 as well. I'm not really familiar with this new inline suggestion mechanism either, but it appears to not allow further changes while reviewing.